### ISSUE TITLE
[Feature] Add MQTT X in Debian repository

### ISSUE THREAD

**Issue by InfoLibre** [emqx/MQTTX#1169]
Could you add MQTT X in Debian repository? Then, it could be available in Ubuntu and Linux Mint too.

**Comment by ysfscream** [comment]
Hi, These should be the Debian packages?
<img width="1439" alt="image" src="https://user-images.githubusercontent.com/21299158/208307392-49fde712-30bf-4d98-97db-5deb7bba67a7.png">

**Comment by InfoLibre** [comment]
Yes, but I'm not sure you can add it as it is. Find how to do thta.

https://wiki.debian.org/HowToPackageForDebian

https://www.debian.org/doc/manuals/distribute-deb/distribute-deb.html

Etc.

**Comment by ysfscream** [comment]
These are the targets of the installers I can currently build for Linux.

<img width="660" alt="image" src="https://user-images.githubusercontent.com/21299158/208802797-a6587f74-fae7-43aa-85d4-e0e94e17a392.png">
